packages/workspace/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@nrwl/workspace",
   "version": "0.0.1",
   "description": "Extensible Dev Tools for Monorepos",
+  "scripts": {
+    "postinstall": "rimraf ./node_modules/listr/node_modules/rxjs"
```

Comment:
Is `rimraf` guaranteed to be installed on the machine?  I don't see it in `devDependencies`.

Reply:
There are a bunch of underlying dependencies that use it, but we could add it as an explicit dependency